virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc

```diff
@@ -10,13 +10,20 @@ You can build a virtual machine image into a container disk and store it in your
 include::modules/virt-about-container-disks.adoc[leveloffset=+1]
 include::modules/virt-preparing-container-disk-for-vms.adoc[leveloffset=+1]
 
+[IMPORTANT]
```

Comment:
IMO, this would be more visible towards the top of the assembly.

Reply:
Moved it up after the intro.